source/app/utils/document.js

```diff
@@ -307,7 +308,7 @@ export function resetFormsonPage() {
 export function getPageEntityPairs(document, pageNumber,comprehendService) {
   // Get all blocks of Entities for a PAGE
   let blocks = []
-  if (comprehendService == COMPREHEND_MEDICAL_SERVICE){
+  if (comprehendService == COMPREHEND_MEDICAL_SERVICE && ENABLE_COMPREHEND_MEDICAL){
```

Comment:
I don't think this is the right approach. We should avoid calling this function if comprehend medical is not enabled. This function is only used in `view.js`:
```js
const docData = useMemo(() => {
    const pairs = getDocumentKeyValuePairs(document)
    const tables = getDocumentTables(document)
    const lines = getDocumentLines(document)
    const entities = getDocumentEntityPairs(document, COMPREHEND_SERVICE)
    const medicalEntities = getDocumentEntityPairs(document, COMPREHEND_MEDICAL_SERVICE)
    return { pairs, tables, lines, entities , medicalEntities }
    // eslint-disable-next-line

  }, [document, document.textractResponse ,document.medicalComprehendResponse, document.comprehendResponse])
```

The way it's done in this PR, `entities` and `medicalEntities` are both going to be the same thing when comprehend medical is disabled. This is not exactly what we want. I would prefer if `docData` was missing the `medicalEntities` key altogether when this feature is disabled.

Reply:
Will follow up offline. Have a query regarding this.